.circleci/config.yml

```
@@ -218,10 +229,13 @@ jobs:
             - v1-bazel-cache-core-{{ .Branch }}-{{ .Revision }} 
             - v1-bazel-cache-core-{{ .Branch }} 
             - v1-bazel-cache-core-main 
+            - gem-v1-{{ arch }}-{{ .Branch }}-{{ checksum "Gemfile.lock" }}
+            - gem-v1-{{ arch }}-main-{{ checksum "Gemfile.lock" }}
 
       - run: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> ~/.npmrc
       - run: echo -e $GPG_KEY | gpg --import --batch
-      - run: npx auto shipit --only-graduate-with-release-label -vv
+      - run: source ~/.bashrc && bundle install
+      - run: source ~/.bashrc && CIRCLE_BUILD_NUMBER=$(cat .ios-build-number) npx auto shipit --only-graduate-with-release-label -vv
```

You might be able to merge this into 1 command 

```suggestion
      - run: |
            source ~/.bashrc
            bundle install
            export CIRCLE_BUILD_NUMBER=$(cat .ios-build-number)
            npx auto shipit --only-graduate-with-release-label -vv
```